var/spack/repos/spack_repo/builtin/packages/py_holoviews/package.py

```diff
@@ -14,8 +14,8 @@ class PyHoloviews(PythonPackage):
 
```

Comment:
You can put me down as maintainer for holoviews if you like

Reply:
added